src/proj_functions.jl

```diff
@@ -0,0 +1,211 @@
+@doc "Return true if the projection is a geographic coordinate system" ->
+is_latlong(proj::Projection) = _is_latlong(proj.rep)
+
+@doc "Return true if the projection is a geocentric coordinate system" ->
+is_geocent(proj::Projection) = _is_geocent(proj.rep)
+
+@doc "Return true if the projection is a geocentric coordinate system" ->
+is_identical(p1::Projection, p2::Projection) = _compare_datums(p1.rep, p2.rep)
```

Comment:
oh sure, i should stick to the original name `compare_datums` then. Copied wrongly from https://github.com/OSGeo/proj.4/blob/5d2af3a89be0898458e4f0fe272affc36642d304/src/pj_transform.c#L448-L453